### PR TITLE
ROU-10753: DropdownSearch and DropdownTags blurry issue.

### PR DESCRIPTION
**Pull request by joselrio**
This PR is will fix an issue that was causing Dropdown Search and Tags get blurry when browser zoom in and out was in use.

> Before:
Screenshot 2024-06-06 at 11 18 22

> After:
Screenshot 2024-06-06 at 11 18 32

### CODE REVIEW

File: src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss [hunk 1]
```diff
@@ -379,10 +379,16 @@
 
 // Wrapper options container
 .vscomp-dropbox {
+	--border-color: var(--color-neutral-5);
+	--border-size: 1px;
```
Comment (BenOsodrac) [inline src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss:383]: var(--border-size-s)

File: src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss [hunk 2]
```diff
@@ -379,10 +379,16 @@
...
+
 	background-color: var(--color-neutral-0);
 	border-radius: var(--border-radius-soft);
-	border: var(--border-size-s) solid var(--color-neutral-5);
+	/* use inner shadow to workaround the border issue that causes content to be blurry */
+	box-shadow: inset var(--border-color) 0px 0px 0px var(--border-size);
```
Comment (BenOsodrac) [inline src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss:388]: can't we remove the `px`? improve the performance a tiny bit

**Review by BenOsodrac — CHANGES_REQUESTED** [review]

File: src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss (hunk 1 above)
Comment (joselrio) [reply src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss]: Done!

File: src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss (hunk 2 above)
Comment (joselrio) [reply src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss]: Done!

**Review by BenOsodrac — APPROVED** [review]

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Pull request merged**